models.py

```diff
@@ -29,4 +29,4 @@ def validate_rating(self, key, value):
         return value
 
     def __str__(self):
-        return f"{self.restaurant.name} ({self.review.date:%x})"
+        return f"{self.user_name}: {self.review_date:%x}"
```

Comment:
In SQLAlchemy, a ForeignKey like `restaurant` is actually just an integer, not a fully resolved object, so `restaurant.name` results in an Attribute error. I changed it to reference `user_name` instead, since that's a string. I also fixed `review.date` to `review_date`, which is a bug I introduced in the style PR.